### DISCUSSION TITLE
roboshop || shipping component is not working as expected

### DISCUSSION THREAD

**Discussion by prasanthbangs2016** [learndevopsonline/learndevopsonline#1964] [Q&A]
After provisioning all the components i have tried shipping from given catalogue, after i have entered country location, unable to confirm the shipping.

![image](https://github.com/user-attachments/assets/a6b655d1-efd0-46b6-b9f2-708472a3aece)

i have checked shipping service status and service file , config looks good,it is running and restarted as well however it is not working,

and, other i have mysql is running or no, it is running and restarted too.

shipping ip : 3.83.44.212
mysql ip: 54.152.218.87
workstation ip: 54.167.100.79

**Comment by chaithanya1812** [comment]
![image](https://github.com/user-attachments/assets/ddad4a0b-fbfc-4293-bf96-173c52043bf6)

## I added it now it's working
![image](https://github.com/user-attachments/assets/70efe2a3-bd7e-4622-89bf-e25e84c7346f)

**Comment by prasanthbangs2016** [comment]
Thanks it is working.
